### PR TITLE
add: pet

### PR DESCRIPTION
**Pull request by Owen-sz**

### CODE REVIEW

**Comment by Owen-sz** [comment]
Why is pet not queued to build on ARM?

File: anda/tools/open-in-mpv/update.rhai [hunk 1]
```diff
@@ -1 +1 @@
-rpm.version(gh("Baldomo/open-in-mpv"));
+rpm.version(gh("Baldomo/open-in-mpv"));
```
Comment (madonuko) [inline anda/tools/open-in-mpv/update.rhai]: please revert this

File: anda/tools/pet/pet.spec [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ 
+%description
+%summary
+
+%prep
+%autosetup -n pet-%version
+ 
+%build
+%global debug_package %{nil}
+go build -gcflags "all=-N -l" -ldflags "-linkmode=external -extldflags=-Wl,--build-id" -o pet
```
Comment (madonuko) [inline anda/tools/pet/pet.spec]: Why?

File: anda/tools/open-in-mpv/update.rhai (hunk 1 above)
Comment (Owen-sz) [reply anda/tools/open-in-mpv/update.rhai]: Oops that was a mistake

File: anda/tools/pet/pet.spec (hunk 2 above)
Comment (Owen-sz) [reply anda/tools/pet/pet.spec]: Had a debug package issue

`
mock    | DEBUG: Processing files: Pet-debugsource-0.9.0-1.fc40.x86_64
mock    | DEBUG: RPM build errors:
mock    | DEBUG: error: Empty %files file /builddir/build/BUILD/pet-0.9.0/debugsourcefiles.list
mock    | DEBUG:     Empty %files file /builddir/build/BUILD/pet-0.9.0/debugsourcefiles.list
mock    | DEBUG: Child return code was: 1
`

As for the build ID workaround, that was set early on when I needed it, thought it was still relevant but I guess it builds just fine without it now, will remove it

Comment (korewaChino) [reply anda/tools/pet/pet.spec]: just do %define debug_package %nil

Comment (Owen-sz) [reply anda/tools/pet/pet.spec]: I did in the build section, this build ID thing was a previous error.

**Review by madonuko — APPROVED** [review]

**Comment by raboneko** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|f39|[](https://github.com/terrapkg/packages/pull/2131)|
|✅|f40|[](https://github.com/terrapkg/packages/pull/2132)|
|✅|f41|[](https://github.com/terrapkg/packages/pull/2133)|
|✅|el10|[](https://github.com/terrapkg/packages/pull/2134)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport) and see the [Github Action logs](https://github.com/terrapkg/packages/actions/runs/11169064297) for details

**Pull request merged**